<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nadaj Przesyłkę, PESEL: 961009xxxxx</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="layout_main">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layout_main_columns">
        <item>
         <layout class="QVBoxLayout" name="layout_column_left">
          <item>
           <widget class="QGroupBox" name="groupBox_parcel_type">
            <property name="title">
             <string>Rodzaj przesyłki</string>
            </property>
            <layout class="QVBoxLayout" name="layout_groupBox_parcel_type">
             <item>
              <widget class="QRadioButton" name="radioButton_pocztowka">
               <property name="text">
                <string>Pocztówka</string>
               </property>
               <attribute name="buttonGroup">
                  <string notr="true">buttonGroup_parcel_type</string>
                 </attribute>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_List">
               <property name="text">
                <string>List</string>
               </property>
               <attribute name="buttonGroup">
                  <string notr="true">buttonGroup_parcel_type</string>
                 </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_Paczka">
               <property name="text">
                <string>Paczka</string>
               </property>
               <attribute name="buttonGroup">
                  <string notr="true">buttonGroup_parcel_type</string>
                 </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_check_price">
            <property name="text">
             <string>Sprawdź cenę</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_img_price">
            <item>
             <widget class="QLabel" name="label_img">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>../../../../Downloads/494807_v9_bd.jpg</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_price">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Cena: 1 zł</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_column_right">
          <item>
           <widget class="QGroupBox" name="groupBox_address_data">
            <property name="title">
             <string>Dane adresowe</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_address">
             <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignTop">
              <widget class="QLabel" name="label_street">
               <property name="text">
                <string>Ulica z numerem</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignTop">
              <widget class="QLineEdit" name="lineEdit_street"/>
             </item>
             <item row="2" column="0" alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_postal_code">
               <property name="text">
                <string>Kod pocztowy</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" alignment="Qt::AlignLeft">
              <widget class="QLineEdit" name="lineEdit_postal_code"/>
             </item>
             <item row="4" column="0" alignment="Qt::AlignTop">
              <widget class="QLabel" name="label_city">
               <property name="text">
                <string>Miasto</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" alignment="Qt::AlignTop">
              <widget class="QLineEdit" name="lineEdit_city"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_apply">
        <property name="text">
         <string>Zatwierdź</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
